packages/components/icons/README.md

```diff
@@ -42,3 +42,11 @@ Main use cases are:
     label={this.props.intl.formatMessage(messages.exportList)}
   />
   ```
+
+## Adding New Icons to UI-Kit
+
+1. Add the icon's SVG to `packages/components/icons/src/raw-components` directory. Please follow the style conventions of existing icon files. File names should be the icon's name in kebab-case (e.g., view-grid-plus.js).
```

Comment:
Hmm the first step is to add the SVG file to the `src/svg` folder. Then you simply need to run the `generate-icons` script.

That should be it if I'm not mistaken. The components are automatically generated.